lib/store/pocketbase.dart

```diff
@@ -16,6 +16,13 @@ class PocketBaseService {
     return list.items.map((e) => e.getStringValue('uid')).toList();
   }
 
+  void _subscribeToRealtime(String collection) {
+    pb.collection(collection).subscribe('*', (e) {
+      // Handle the event, for example, refresh your data
+      _getModels(collection);
+    });
+  }
+
```

Comment:
why is this change in this branch? This branch is supposed to be only for the visual bug fix for #56

Reply:
which branch is this coming from?

Reply:
Hello @lukas-h, it was coming from the previous branch. it was 3 weeks past changes. so I have not made any changes on file. I have moved this files.